### ISSUE TITLE
More event notifications

### ISSUE THREAD

**Issue by CodedGold** [JonnyPtn/zomboi#36]
Hello! I'm sorry if I put this in the wrong place.

I have some suggestions that would help me and my staff better manage our community.

1) Tracking Vehicle logs: When someone takes engine parts, smashes a vehicle window, hotwiring, taking gasoline, taking a vehicle part.

Mainly, this is for documentation to prevent griefing. Allowing these logs to be generated in the discord would help my staff team follow up and administer corrective actions if someone went against our rules surrounding vehicle ownership. It could be something as simple as "[Username] hotwired a vehicle at [location coordinates]!" "[Username] took an engine part from [location coordinates]!"

2) Tracking Destruction logs: When someone dissembling / destroys something.

We don't want players grinding skills using the residential communities in game (like destroying furniture, showers, sinks, fridges, etc). Need these logs generated to make sure they aren't doing this. Same as above: "[Username] dissembled something at [location coordinates]!" "[Username] destroyed something at [location coordinates]!"

3) Tracking Admin logs: When someone spawns an item.

This is pivotal to ensuring admins aren't abusing their access. "[Username] spawned [item]!" or "[Username] spawned an item!"

Thanks for looking at this and looking forward to the help!

**Comment by voidshogun** [comment]
I totally agree with the addition of this suggestion..
that would help server staff better manage their community.

**Comment by JonnyPtn** [comment]
If it's logged to a log file by the game then it should be possible

**Comment by CodedGold** [comment]
Most, if not all, of these things are logged to a log file, I believe. I'm just not sure how to script it. :/

**Comment by petegilb** [comment]
I’d be happy to take a look into adding these things. Just gonna be a bit since I’m in the process of moving right now

**Comment by CodedGold** [comment]
Looking forward to it @petegilb!!! Thank you for the help with this! Also, hope your move goes smooth!

**Comment by petegilb** [comment]
@CodedGold I've started looking at this and it looks like the _cmd.txt and _map.txt logs are out best bet for this. Would you mind sharing your log folder so I can look for more data instead of going into the game and manually doing different things? Not sure if there's any sensitive info in there so maybe would be best to share privately rather than just in this issue here too. lmk!

**Comment by CodedGold** [comment]
@petegilb I can send it to you, sure! Honestly not sure if I can send a message to you on github so if you're comfortable using discord, my ID is CodedGold :)

**Comment by petegilb** [comment]
Awesome I added you! I'm thinking that it also might be best to have some kind of config file other than a .env file since we're gonna have so many options, so I might write a ConfigParser .ini file for the non-sensitive bot tokens and stuff like that.

**Comment by JonnyPtn** [comment]
Yeah I did consider config files at the start but considering for server stuff like this a lot of times it's easier to set environment variables (e.g as part of a docker config). Still always open minded if there's a nice solution to work with both

**Comment by petegilb** [comment]
Made a basic admin log handler that posts the logs from the other logs to a separate channel. Could do more filtering and clean up but it's functional for now. Not sure how performant it is but it was working alright for me. Might spam a bit tho lol. https://github.com/petegilb/zomboi/tree/issue36
